samples/drivers/watchdog/sample.yaml

```diff
@@ -48,7 +48,18 @@ tests:
   sample.drivers.watchdog.stm32h7_wwdg:
     extra_args: DTC_OVERLAY_FILE=boards/stm32h7_wwdg.overlay
     filter: dt_compat_enabled("st,stm32-window-watchdog")
-    platform_allow: nucleo_h743zi
+    platform_allow:
```

Comment:
isnt this already covered with the filter above? why the platform allow thing?

Reply:
also, please add integration_platforms with one platform for PR verification.

Reply:
One typical example where https://github.com/zephyrproject-rtos/zephyr/issues/81458 would help.